asset/quickstart/export-assets/main.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Google LLC
```

Comment:
Would you mind adding a test? You can follow this example for how to set up the test, set the `GOOGLE_CLOUD_PROJECT` environment variable, and capture output printed to standard out: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/jobs/v3/quickstart/main_test.go.

Reply:
It's said that "quickstarts contain no testable code" (https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/policies/testing.md#requirements), I thought we don't need to write it as I see from pubsub's quickstart here: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/pubsub/pubsub_quickstart/main.go?

Reply:
Also there's some setup for the project before running the main, unless we have a fixed test project for golang-samples default project, otherwise even main won't be run successfully.

Reply:
I'll see if we can add a user journey test here, will have to give up if the project setup effort is too expensive.

Reply:
Added.